udf/deanonymize_event.sql

```diff
@@ -0,0 +1,43 @@
+/*
+
+Rename struct fields in anonymous event tuples to meaningful names
+
+*/
+CREATE TEMP FUNCTION
+  udf_deanonymize_event(tuple STRUCT<
+    f0_ INT64,
+    f1_ STRING,
+    f2_ STRING,
+    f3_ STRING,
+    f4_ STRING,
+    f5_ ARRAY<STRUCT<key STRING, value STRING>>>) AS (
+    (SELECT
+      AS STRUCT tuple.f0_ AS `timestamp`,
```

Comment:
this should use `STRUCT(` instead of `(SELECT AS STRUCT`, because BigQuery can only handle a limited number of expression subqueries.